nautobot/dcim/navigation.py

```diff
@@ -227,17 +227,17 @@
                         ),
                     ),
                     NavMenuItem(
```

Comment:
Were we going to move `Device Families` above `Manufacturers` in the nav as a part of this PR, or is that still up in the air? @lampwins